app/src/main/java/com/example/weatherapp/Api.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.weatherapp
+
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface Api {
+    @GET("weather")
+    fun getCurrentConditions(
+        @Query("zip") zip: String = "55411",
+        @Query("units") units: String = "imperial",
+        @Query("appid") appId: String =  "0e3745b8b10701ce9dc29c580c91b68f"
+    ) : Call<CurrentConditions>
+
+    @GET("daily")
```

Comment:
should be `@GET("forecast/daily")` so you can have the same base url in your retrofit instances.